src/util/BaseLoginController.js

```diff
@@ -92,8 +94,14 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
     toggleButtonState: function (state) {
       var button = this.$el.find('.button');
       button.toggleClass('link-button-disabled', state);
-    }
+    },
 
+     // Event triggered after a page is rendered along with the classname to identify the page
+    triggerEvent: function() {
+      _.defer(_.bind(function () {
```

Comment:
It turns out that the way we're now invoking things no longer needs the defer, so we can remove it. The animation sequence attaches both controllers to the dom, and does some transition stuff. When we call it after render/success, we are guaranteed that it's attached to the dom and there are no other controllers.